hs_core/templates/pages/genericresource.html

```diff
@@ -1587,9 +1587,11 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             <div class="tab-pane" id="relevantTools">
                             {% if not metadata_form %}
                                 {% for tool in relevant_tools %}
-                                    <h4>{{ tool.title }}</h4>
-                                    <a href="{{ tool.url }}" target="_blank"><button class="btn btn-primary">Launch App</button></a>
-                                    <hr>
+                                    <div style="display:inline-block; padding-left:20px; text-align:center;">
+                                        <h4>{{ tool.title }}</h4>
+                                        <a href="{{ tool.url }}" target="_blank"><img src="{{ tool.icon }}" style="width:100px; height:100px;"
```

Comment:
@shawncrawley hydroshare_core.css is located in hydroshare code base under theme/static/css/hydroshare_core.css. If there are more styles specifically related to this webapp, I'd recommend to make your own css under your_app/static/css directory, then include the css in the template file that is referring the css file by including it like "<link href=""{{ STATIC_URL }}css/your_css_file_name.css" rel="stylesheet"/>. This works since all static files are collected during hydroshare build into {{ STATIC_URL}}css directory